packages/celotool/src/lib/geth.ts

```diff
@@ -915,6 +915,8 @@ export async function startGeth(
     'extip:127.0.0.1',
     '--allow-insecure-unlock', // geth1.9 to use http w/unlocking
     '--gcmode=archive', // Needed to retrieve historical state
+    '--istanbul.blockperiod',
+    blocktime.toString(),
```

Comment:
If `gethConfig.genesisConfig.blocktime` is undefined, do we still want the default to be 1s (set on line 890)?

Reply:
Yeah I think it's fine for now since this is mostly used for e2e tests